lib/mongoose_push_web/plugs/maybe_put_swagger_ui.ex

```diff
@@ -0,0 +1,30 @@
+defmodule MongoosePushWeb.Plug.MaybePutSwaggerUI do
+  import Plug.Conn
+  @behaviour Plug
+
+  @impl Plug
+  def init(opts) do
+    enable =
+      Application.get_env(:mongoose_push, :openapi)[:expose_spec] &&
+        Application.get_env(:mongoose_push, :openapi)[:expose_ui]
+
+    opts =
+      if enable do
+        OpenApiSpex.Plug.SwaggerUI.init(opts)
+      end
+
+    %{enabled: enable, opts: opts}
+  end
+
+  @impl Plug
+  def call(conn, %{enabled: enable, opts: opts}) when enable == false or opts == nil do
```

Comment:
Why do we check `opts` here? Is that documented that `nil` is a failure scenario? 
Actually, even if it is, the plug that generates this value should be responsible for handling it.

Reply:
Also, there is not need for guard, let's just match on `enabled: false`.

Reply:
I've just pushed the update you're asking for :D